src/models/index.ts

```diff
@@ -117,6 +118,7 @@ export interface ISelectField {
 	size: any;
 	hidden?: boolean;
 	displayName?: string;
+	required?: boolean;
 }
 
```

Comment:
Please remove the use of two similar interfaces

Reply:
I have already made changes in 17 files and this change will make change in 10 - 15 file, I will make seprate pr for that